src/CommandingOfficers/AW3/AW3ValueFinders.java

```diff
@@ -0,0 +1,73 @@
+package CommandingOfficers.AW3;
+
+import CommandingOfficers.Commander;
+import CommandingOfficers.IValueFinder;
+import Engine.UnitActionFactory;
+import Units.Unit;
+import Units.UnitModel;
+
+public abstract class AW3ValueFinders
+{
+  public static class CostValueFinder implements IValueFinder
+  {
+    int maxDamage = 3;
+    @Override
+    public int getValue(Commander attacker, Unit unit)
+    {
+      // Ignore Oozium, if we ever add those
+      int health = unit.getHealth();
+      if( health < 10 )
+        return 2;
+      health = Math.min(health, maxDamage * 10);
+
+      int value = unit.getCost() * health / UnitModel.MAXIMUM_HEALTH;
+
+      if( !unit.CO.isEnemy(attacker) )
+        return -value;
+      return value;
+    }
+  }
+  public static class HealthValueFinder implements IValueFinder
```

Comment:
Sturm's is called `HPValueFinder`. Minor nit.

Reply:
And if you look closely, you'll see he uses HP instead of health.

Reply:
I didn't look closely. I assumed that two functions to do apparently the same job would work the same way. Why don't they work the same way?

Reply:
Because AW3's targeting has special casing for units under 1.0 HP.